### DISCUSSION TITLE
Modern Docking in the wild (share yours!)

### DISCUSSION THREAD

**Discussion by i-make-robots** [andrewauclair/ModernDocking#153] [Show and tell]
Showcasing a bit of what ModernDocking can do.

![image3](https://github.com/andrewauclair/ModernDocking/assets/1464454/ea8fc16a-0bb9-48ad-8a0a-d3e40a30066e)

**Comment by i-make-robots** [comment]
I wanted a docking system for about a decade and never took the plunge until I found ModernDocking.  
10/10 support, the No. 1 deciding factor for me.  Dude, you rock.  Point me to the website for this project.  Is there a patreon or something so I can buy you a beverage?

**Reply by andrewauclair** [reply]
Thanks! Really appreciate the kind words :) I made this library purely because I hated all the existing docking frameworks, especially the one I was using (FlexDock). I'm glad it has been useful for other people despite Java Swing being old and docking being a niche feature.

There's no website currently, but I do own andrewauclair.com, which redirects to my "blog" on andrewauclair.github.io and that is completely empty. The Jetbrains Writerside tool that I'm using for the new documentation can generate a website. I'm really hoping that I can put that on my github.io, but haven't tried it yet.

I just setup a Buy Me a Coffee, you can find it here: https://www.buymeacoffee.com/andrewauclair.

**Reply by i-make-robots** [reply]
Java Swing is old but that's not a pejorative.  It's well tested and documented!  I was fighting the urge to write a docking system until I found this.  Fits the bill perfectly.  A real delight to work with.
